inst/qml/raincloudPlots.qml

```diff
@@ -0,0 +1,900 @@
+
+							"<br><br>" +
+
+							"<h3>Element Settings</h3>" +  // All of this here, because infoLabel{} for DoubleField{} does not work.
+
+							"<h4>Nudge</h4>" +
+							"The nudge for violin, box, and point determines how far the elements are nudged from the center (axis tick).<br>" +
+							"By default, the box is in the center (nudge = 0) and the violin is nudged to the right. " +
+							"The points are nudged to the left of the box.<br>" +
+							"With a custom orientation (see Advanced section), the points get centered (nudge fixed to 0). Violin and box are nudged depending on orientation: left or right." +
+
+							"<h4>Height, Width, and Spread</h4>" +
+							"These settings determine the respective properties of the elements:<br><br>" +
+							"How high should the violin peak? (Does not change the proportions of the probability density.)<br>" +
+							"With a custom orientation to the left (see Advanced Section), it can be " +
+							"helpful to decrease the height if the violin overlaps with axis ticks.<br><br>" +
+							"How wide should the box be?<br><br>" +
+							"How wide should the points spread? I.e., how jittered should they be along the x-axis (y-axis if horizontal plot?.<br>" +
+							"Note, that this still correctly shows the values of the points on the dependent variable axis.<br>" +
+							"(If you would also like y-jitter see the 'Jitter' option for points.)" +
+
+							"<h4>Box Padding</h4>" +
+							"With a secondary factor, there will be a box for each factor level. With the padding, you can change the spacing between theses boxes.<br>" +
+							"Increasing the padding will - visually - decrease the box width. If you want to keep box width constant, also increase the box width." +
+
+							"<h4>Point Size</h4>" +
+							"We recommend a greater size, the fewer points you have." +
+
+							"<h4>Opacity</h4>" +
+							"Increase this to _decrease_ the transparency of the respective element." +
+
+							"<h4>Outline and Outline Width</h4>" +
+							"Would you like the outline to match the 'Color palette', be 'black', or have 'none' at all?<br>" +
+							"The width determines how thick the outline is." +
+
+							"<h4>Violin Smoothing</h4>" +
+							"This percentage determines the smoothness of the probability density.<br>" +
+							"The lower, the stronger it is influenced by the presence/absence of individual points." +
+
+							"<h4>Point Jitter</h4>" +
+							"Use this with care!<br>" +
+							"It slightly jitters the points along the dependent variable axis. This means that the position of the points no longer matches the values of the observations.<br>" +
+							"If you want to have less overlap between the points while keeping an accurate visualization, increases the point spread.<br>" +
+							"Using point jitter can be useful in some cases. For example, if you are working with likert data." +
+
+							"<h4>ID Lines Width</h4>" +
```

Comment:
```suggestion
			info: 		qsTr(
							"Whether or not the points of a cloud should be shown. If un-checked, opacity is set to 0.<br>\n"
							"If you have many, many points, it can be helpful to hide them.\n"

							"<br><br>\n"

							"<h3>Element Settings</h3>\n"  // All of this here, because infoLabel{} for DoubleField{} does not work.

							"<h4>Nudge</h4>\n"
							"The nudge for violin, box, and point determines how far the elements are nudged from the center (axis tick).<br>\n"
							"By default, the box is in the center (nudge = 0) and the violin is nudged to the right. \n"
							"The points are nudged to the left of the box.<br>\n"
							"With a custom orientation (see Advanced section), the points get centered (nudge fixed to 0). Violin and box are nudged depending on orientation: left or right.\n"

							"<h4>Height, Width, and Spread</h4>\n"
							"These settings determine the respective properties of the elements:<br><br>\n"
							"How high should the violin peak? (Does not change the proportions of the probability density.)<br>\n"
							"With a custom orientation to the left (see Advanced Section), it can be \n"
							"helpful to decrease the height if the violin overlaps with axis ticks.<br><br>\n"
							"How wide should the box be?<br><br>\n"
							"How wide should the points spread? I.e., how jittered should they be along the x-axis (y-axis if horizontal plot?.<br>\n"
							"Note, that this still correctly shows the values of the points on the dependent variable axis.<br>\n"
							"(If you would also like y-jitter see the 'Jitter' option for points.)\n"

							"<h4>Box Padding</h4>\n"
							"With a secondary factor, there will be a box for each factor level. With the padding, you can change the spacing between theses boxes.<br>\n"
							"Increasing the padding will - visually - decrease the box width. If you want to keep box width constant, also increase the box width.\n"

							"<h4>Point Size</h4>\n"
							"We recommend a greater size, the fewer points you have.\n"

							"<h4>Opacity</h4>\n"
							"Increase this to _decrease_ the transparency of the respective element.\n"

							"<h4>Outline and Outline Width</h4>\n"
							"Would you like the outline to match the 'Color palette', be 'black', or have 'none' at all?<br>\n"
							"The width determines how thick the outline is.\n"

							"<h4>Violin Smoothing</h4>\n"
							"This percentage determines the smoothness of the probability density.<br>\n"
							"The lower, the stronger it is influenced by the presence/absence of individual points.\n"

							"<h4>Point Jitter</h4>\n"
							"Use this with care!<br>\n"
							"It slightly jitters the points along the dependent variable axis. This means that the position of the points no longer matches the values of the observations.<br>\n"
							"If you want to have less overlap between the points while keeping an accurate visualization, increases the point spread.<br>\n"
							"Using point jitter can be useful in some cases. For example, if you are working with likert data.\n"

							"<h4>ID Lines Width</h4>\n"
```

Reply:
I dont think you need the `+` everywhere.

Reply:
I just tried dropping the `+` but then the GUI won´t load. So I will leave it in, okay?

Reply:
Ah, I originally only removed the `+`s but not the quotes at the end of each line!
Your commit works for me now, however I was not able to change the indentation without also getting a messy layout in the documentation (see pictures).
Is there a way to fix this? Otherwise I would prefer `+` and `"` at the end of each line so that it looks tidier (at least in my vscode).

<img width="1104" alt="Screenshot 2024-04-05 at 19 24 23" src="https://github.com/jasp-stats/jaspDescriptives/assets/123652963/78f55f2e-fc67-4e82-986e-fbb6d63f867e">
<img width="1046" alt="Screenshot 2024-04-05 at 19 24 40" src="https://github.com/jasp-stats/jaspDescriptives/assets/123652963/b34f6abe-5c05-4507-945d-1ad93f3da0fa">

Reply:
Just remove the leading spaces...

Reply:
`shift+tab` is your friend ;)

Reply:
So you mean like this?
<img width="904" alt="Screenshot 2024-04-07 at 20 33 49" src="https://github.com/jasp-stats/jaspDescriptives/assets/123652963/532c0db8-e710-4639-a16f-13f741356b97">

with documentation like:

<img width="1099" alt="Screenshot 2024-04-07 at 20 34 25" src="https://github.com/jasp-stats/jaspDescriptives/assets/123652963/3f895723-bea9-4cd3-ac05-ee9b755e1573">

I do not like it, as I said above:
Without the leading spaces, the code is not indented. However, I would like it indented.
> Is there a way to fix this?
> Otherwise I would prefer + and " at the end of each line so that it looks tidier

Reply:
Well, I presonally like being able to just easily add and edit lines in a codebase more important than correct indentation. You know, if someone wants to add a paragraph to your `info` they have to do so in strings `\n"+` etc.
With my way you just type...

So this is a convenience thing because Im all for beautiful indentation and spacing in general!

However, im not very likely to be the one to have to work on this so as long as it ends up in the translation files correctly I dont really mind which one you use ;)

Reply:
Ah okay, does it end up in the translation files correctly either way?
How do we check?

If it does, may I revert your commit do go back to `"` and `+`?

Reply:
If it ends up in the *.po files fine of course you can revert it

Reply:
Alright, so can we now pull the rainclouds into the next nightly?
Or is there anything else?

Reply:
That's fine with `+` to concatenate strings and just we need to pay attention to make sure it is in a complete `qsTr()` call.will generate a complete block. in .po file will looks like:
```po
#: nst/qml/raincloudPlots.qml:607
msgctxt "raincloudPlots|"
msgid ""
"Whether to also show an interval around the mean.<br>This hides the box, but "
"if you really want, you can go up and check 'Show box' again.<br><br>You can "
"choose between ±1 standard deviation or a confidence interval.<br>The "
"confidence interval is computed independently for each group. For this, the "
"corresponding checkbox needs to be checked, to acknowledge that.<br>This "
"computation means that any factors are treated as between-factors.<br>If "
"your data does not meet this assumption, but you still want to show "
"intervals around the means, you can specify custom intervals (see "
"below).<br>One example where this assumption does not hold is with ID input. "
"This is why ID input disables the computation of confidence "
"intervals.<br><h4>Method</h4>How should the confidence interval be "
"computed?<br>'Normal model' uses a standard error defined as: "
"_SD_<sub>group</sub> / square root of _N_<sub>group</sub><br>'T model' "
"yields results identical to a one-sample t-test.<br>For 'Bootstrap' you can "
"further specify the number of samples and set a seed for reproducible "
"results."
msgstr ""
```

Reply:
>Without the leading spaces, the code is not indented. However, I would like it indented.

for indent things maybe we should just add the style rule in .css for help docs. so developer shouldn't add indent manually? what do you think @JorisGoosen

Reply:
Well, the indent is generated by markdown in any case.

Actually, that is a good point. @vincentott is there a reason you are typing html there and not just markdown?
Because I think it is ok as the html will not be stripped by markdown.
That already handles all the indents anyway.

Me and vincent were just talking about the indent in the qml code

Reply:
Once I started using html/css in the .R file for plot and table footnotes, I also kept using it in the .qml file.
So, may I leave it as is?

Reply:
yeah you can